tests/Microsoft.Bot.Builder.Expressions.Tests/ExpressionEngineTests.cs

```diff
@@ -259,9 +259,9 @@ public class ExpressionEngineTests
             Test("join(foreach(items, item, concat(item, string(count(items)))), ',')", "zero3,one3,two3", new HashSet<string>{ "items"}),
 
             //Memory access
-             Test("first(nestedItems).x", 1, new HashSet<string> { "nestedItems"}),
-             Test("property(bag, concat('na','me'))","mybag")
-
+            Test("first(nestedItems).x", 1, new HashSet<string> { "nestedItems"}),
+            Test("property(bag, concat('na','me'))","mybag"),
+            Test("bag['name']","mybag")
```

Comment:
You should add more tests in particular where it is not a constant string.

Reply:
More tests have been added in the below commit